cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=100)
```

Comment:
It's a good practice to add a `help_text` parameter to your model fields. This provides a description of the field that can be used in Django's automatically-generated documentation and forms.